app/models/shopify/webhooks/webhooks_manager.rb

```diff
@@ -0,0 +1,102 @@
+module Shopify
+  module Webhooks
+
+    class WebhooksManager
+      class CreationFailed < StandardError
+      end
+
+      def self.get_manager(params)
```

Comment:
so that I did not have to repeat 
```
        org_uid = params.fetch(:org_uid)
        shop_name = params.fetch(:shop_name)
        token = params.fetch(:token)
```
but maybe the `WebhooksManager` constructor could directly a param.